R/provider-azure.R

```diff
@@ -43,12 +44,12 @@ NULL
 #' @export
 #' @examples
 #' \dontrun{
-#' chat <- chat_azure_openai(deployment_id = "gpt-4o-mini")
+#' chat <- chat_azure_openai(model = "gpt-4o-mini")
 #' chat$chat("Tell me three jokes about statisticians")
 #' }
 chat_azure_openai <- function(
   endpoint = azure_endpoint(),
-  deployment_id,
+  model,
```

Comment:
This change makes `chat_azure_openai()` compatible with `chat()`

Reply:
Nice catch!